services/resource-usage-tracker/tests/unit/with_dbs/conftest.py

```diff
@@ -62,3 +70,75 @@ def mocked_prometheus(mocker: MockerFixture) -> mock.Mock:
         autospec=True,
     )
     return mocked_get_prometheus_api_client
+
+
+@pytest.fixture()
+def user_id() -> UserID:
+    return UserID(randint(1, 10000))
+
+
+@pytest.fixture()
+def user_db(postgres_db: sa.engine.Engine, user_id: UserID) -> Iterator[dict]:
+    with postgres_db.connect() as con:
+        # removes all users before continuing
+        con.execute(users.delete())
+        con.execute(
+            users.insert()
+            .values(
+                id=user_id,
+                name="test user",
+                email="test@user.com",
+                password_hash="testhash",
+                status=UserStatus.ACTIVE,
+                role=UserRole.USER,
+            )
+            .returning(sa.literal_column("*"))
+        )
+        # this is needed to get the primary_gid correctly
```

Comment:
are you sure you need this?
also we already have fixtures to create_users, why not use them?

Reply:
i will add it in the upcoming PR today afternoon/tomorrow